### PR TITLE
Release v0.37.0

### PR DESCRIPTION
**Pull request by Umekawa**

### CODE REVIEW

**Comment by Umekawa** [comment]
This pull request will merge to master after merge https://github.com/increments/qiita-markdown/pull/94

**Comment by Umekawa** [comment]
@mishiwata1015 please review :pray:

**Review by mishiwata1015 — APPROVED** [review]

**Pull request merged**
